### DISCUSSION TITLE
Fehler bei Update "Uncaught Error: Class 'Projects_Model_Project' not found" bei update\15.php:471hp

### DISCUSSION THREAD

**Discussion by habreli** [tine20/tine20#7425] [General]
Und zwar von tine-2022.10.08 nach tine-2022.10.22
Konnte das bereits jemand lösen und mir Tipps geben?
Ich verwende PHP 7.4.29 falls das relevant sein sollte.
Danke

**Comment by pschuele** [comment]
hi, denke hier wurde einfach vergessen, das Projects zip/tar file auch zu entpacken. dann wird die klasse auch gefunden.
habe aber trotzdem noch einen kleinen fix dafür eingebaut.

**Reply by habreli** [reply]
Danke - aber für einen unbedarften Menschen.
Das File für Tine-2022.10.22 ist nach wie vor von 2022 - da ist keine Änderung drinnen - ich hab es trotzdem nochmals runtergeladen und verwendet -> keine Änderung.
Wo finde ich denn das erwähnte Projects zip/tar welches zu entpacken vergessen wurde?
Kann ich das auch manuell machen?
Grüße aus Österreich.
